src/components/NotificationsDropdown.tsx

```diff
@@ -164,8 +269,73 @@ export default function NotificationsDropdown(props: NotificationsDropdownProps)
       const dateB = new Date(b.createdTime);
       return dateB.getTime() - dateA.getTime();
     });
+
+    if (timeZoneOrgNotification) {
```

Comment:
we need to think about how to move this outside of this file.. since the number of features with this type of notification will increase over time.. maybe need to have some sort of 'FeatureNotifications' component which is embedded into this component?

Reply:
Yes, I thought about it, this is not scalable for the future. I can iterate after

Reply:
Just so we don't lose track, can we create a ticket for this. I think we may have missed tickets for previous PRs as well (componentize popover menu, etc.)

Reply:
Yes! We have that previous ticket created also

Reply:
Awesome, thanks @constanzauanini